docs/guides/dao-members/proposals.md

```diff
@@ -160,31 +160,42 @@ To create a new proposal using the DAO dashboard:
 
    > Defines the signature of the function to call within the target contract.
    > For the target contract USDC mentioned above use
+
+   <!-- WARNING: A space wil cause the transaction to fail in solidity. -->
+
    > `transfer(address,uint256)`. <span style="color:red">Do not use any spaces
    > in the signature: leading, trailing or otherwise.</span>
 
 7. Enter an **ETH Value**.
 
    > You can use zero if the target function is not `payable`.
 
-8. Enter **Parameters** which are the arguments that will be used to satisfy the
-   signature of the target contract function.
+8. In the case of step #5 above, the address
+   `0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48` is for the USDC contract. You
+   will be calling its `transfer` function, as indicated in step #6, should the
+   proposal pass. Enter **Parameters** which are the arguments that will be used
```

Comment:
I think this step number 8 should begin with "Enter **Parameters** which are the..." then your addition ("In the case of step 5...") should follow

Reply:
That does make sense, changed